lib/tessel/provision.js

```diff
@@ -13,6 +13,16 @@ var directory = path.join(osenv.home(), '.tessel');
 var filename = 'id_rsa';
 var filepath = path.join(directory, filename);
 var remoteAuthFile = '/etc/dropbear/authorized_keys';
+var debug = require('debug')('provision');
+
+Object.defineProperty(Tessel, 'TESSEL_AUTH_KEY', {
```

Comment:
@johnnyman727 it will stay on filepath because this is only called when active called I thought.